src/stores/tenant.rs

```diff
@@ -218,9 +219,15 @@ impl Tenant {
                         &self.apns_team_id,
                     ) {
                         (Some(topic), Some(pkcs8_pem), Some(key_id), Some(team_id)) => {
-                            let mut reader = BufReader::new(pkcs8_pem.as_bytes());
+                            info!("apns token (p8) provider is matched");
+                            // For supporting the legacy format of the p8 key which is double base64 encoded
```

Comment:
If it's "double" base64 encoded, then won't decoding always pass?

Reply:
The updated p8 key format in the database is plain text like:
```
-----BEGIN PRIVATE KEY-----
...base64 encoded key...
-----END PRIVATE KEY-----
```
which is actually the content of the p8 key file.

The legacy format was that content but encoded in base64 with the `BEGIN/END PRIVATE KEY` header and footer.
So the new format should not be decoded successfully (because of the header and footer) and we should pass the plain text instead in `unwrap_or`.

We should encode p12 because it's content is in bytes, but skip for further "double" encoding p8.

Reply:
Resolving this as we've decided to go back to the base64 encoded PEM key for p8.